stock_landed_costs_purchase_auto/models/purchase_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    landed_costs_ids = fields.One2many(
+        comodel_name="stock.landed.cost",
+        inverse_name="purchase_id",
+        string="Landed Costs",
+    )
+    landed_costs_number = fields.Integer(compute="_compute_landed_costs_number")
+
+    def _compute_landed_costs_number(self):
+        for item in self:
+            item.landed_costs_number = len(item.landed_costs_ids)
+
+    def _prepare_landed_costs_values(self):
```

Comment:
Sorry, I slipped out an `s` in between:

```suggestion
    def _prepare_landed_cost_values(self):
```